docs/source/runtime.rst

```diff
@@ -319,66 +320,52 @@ For more on SLURM configuration, please see the `slurm.conf manual
-:doc:`geopmlaunch(1)<geopmlaunch.1>` man page.
-
-The ``geopmlaunch`` command supports various underlying MPI application
-launchers as shown in the :doc:`geopmlaunch(1)<geopmlaunch.1>` man page. If
-your system's launch mechanism is not supported, then you must enforce affinity
-requirements, and all options to the GEOPM runtime must be passed through
-environment variables. Please consult the :doc:`geopm(7)<geopm.7>` man page for
+application and the GEOPM control thread on each compute node.  This wrapper is
+documented in the :doc:`geopmlaunch(1)<geopmlaunch.1>` man page.
+
+If your system's launch mechanism is not supported then options to the GEOPM
+runtime must be passed through environment variables and some features of the
+``geopmlaunch`` command (such as process CPU affinity management) will not be
+available.  Please consult the :doc:`geopm(7)<geopm.7>` man page for
 documentation of the environment variables used by the GEOPM runtime that would
-otherwise be controlled by the wrapper script.
+otherwise be controlled by the wrapper script and see :ref:`overview:Profiling Applications without ``geopmlaunch```
+for details.
 
 CPU Affinity Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The GEOPM runtime requires each of the application's MPI processes to
-be affinitized to different CPUs. This is a critical requirement for
-the runtime and must be enforced by the MPI launch command. When using
-the ``geopmlaunch`` wrapper, these affinity requirements are handled by
-``geopmlaunch`` when the ``--geopm-affinity-enable`` command-line option
-is provided (see :doc:`geopmlaunch(1)<geopmlaunch.1>`). Otherwise, users
-must explicitly affinitize their application using the appropriate options
-for their chosen launcher.
+When using the ``geopmlaunch`` wrapper, the user may optionally provide the
+``--geopm-affinity-enable`` command-line option is provided (see
+:doc:`geopmlaunch(1)<geopmlaunch.1>`). This will enable hardware metrics to be
+more accurately measured on a per-application-region basis by restricting
+process migration.
 
 While the GEOPM control thread connects to the application it will
 automatically affinitize itself to the highest indexed core not used by the
 application if the application is not affinitized to a CPU on every core. If
 the application is using all cores of the system, the GEOPM control thread
 will be pinned to the highest logical CPU.
 
-Many ways exist to launch an MPI application, and no single uniform
-way of enforcing MPI rank CPU affinities can work across all job launch
-mechanisms. OpenMP runtimes, which are linked with compiler choice, also have
-different mechanisms for affinitizing OpenMP threads within CPUs available
-to each MPI process. The GEOPM control thread can also be launched as an
-application thread or process that can either be part of the primary MPI
-application or a completely different MPI application. Due to these factors,
```

Comment:
All of these features still exist in the Controller and geopmlaunch...
why remove this now?

Reply:
Some of these details should be moved to the non-existent HPC tuning guide and/or references to geopmlaunch should be added.